packages/website/docs/docs/quickstart.md

```diff
@@ -46,6 +46,10 @@ function App() {
 
 As well as `BlockNoteView` and `useBlockNote`, we import `@blocknote/core/style.css` to provide default styling for the editor.
 
+::: warning Next.js usage (or other server-side React frameworks)
```

Comment:
I think it might make sense to broaden this to all serverside React frameworks since it's also applicable to those (we've had a couple of people having issues with NX too). We can have the last line be smth like "If you're using NextJS, read our guide on setting it up with BlockNote".

Reply:
NX is a framework for monorepos. My guess is those users were also using nextjs.

Agree that users of other server-side frameworks (e.g.: remix) might run into this as well, but let's wait until we actually get those reports :)